kafka/channel/pkg/dispatcher/dispatcher.go

```diff
@@ -59,41 +59,40 @@ type TopicFunc func(separator, namespace, name string) string
 type KafkaDispatcherArgs struct {
 	ClientID     string
 	Brokers      []string
-	ConsumerMode cluster.ConsumerMode
```

Comment:
@slinkydeveloper  the `consumerMode` is set via this config map:
https://github.com/knative/eventing-contrib/blob/master/kafka/channel/config/400-kafka-config.yaml#L28

please update yaml and remove code that reads it.

Also, this needs to be release noted!

Reply:
there is any crd to update too?